pkg/auth/jwe/manager.go

```diff
@@ -0,0 +1,177 @@
+package jwe
```

Comment:
Seems like this is the same code as `kubernetes/dashboard`, should we refer that code instead of creating our own jwe manager?

Reply:
because we change a bit for the keyHolder interface, replace panic with error return, so I move the code here.